os/booting-on-vultr.md

```diff
@@ -44,7 +44,7 @@ CoreOS is designed to be [updated automatically]({{site.baseurl}}/using-coreos/u
   <div class="tab-content coreos-docs-image-table">
     <div class="tab-pane" id="alpha">
       <div class="channel-info">
-        <p>The alpha channel closely tracks master and is released to frequently. The newest versions of <a href="{{site.baseurl}}/using-coreos/docker">Docker</a>, <a href="{{site.baseurl}}/using-coreos/etcd">etcd</a> and <a href="{{site.baseurl}}/using-coreos/clustering">fleet</a> will be available for testing. Current version is CoreOS {{site.data.alpha-channel.rackspace-version}}.</p>
+        <p>The alpha channel closely tracks master and is released to frequently. The newest versions of <a href="{{site.baseurl}}/using-coreos/docker">Docker</a>, <a href="{{site.baseurl}}/using-coreos/etcd">etcd</a> and <a href="{{site.baseurl}}/using-coreos/clustering">fleet</a> will be available for testing. Current version is CoreOS {{site.alpha-channel}}.</p>
```

Comment:
<p>The alpha channel closely tracks the <code>master</code> branch of the development repository and sees frequent releases. The latest versions of <a href="{{site.baseurl}}/rkt/">rkt</a>, <a href="{{site.baseurl}}/using-coreos/etcd">etcd</a>, <a href="{{site.baseurl}}/using-coreos/docker">Docker</a>, and other essentials are available for testing in the alpha channel. The current alpha version is CoreOS {{site.alpha-channel}}.</p>